test/test_models.py

```diff
@@ -29,3 +29,23 @@ def test_mfcc(self):
         out = model(x)
 
         assert out.size() == (batch_size, num_classes, 2)
+
+
+class TestMelResNet:
```

Comment:
This test does not subclass `unittest.TestCase` so it won't run in `fbcode`.

Reply:
good catch! @jimchen90 -- can you send a follow-up pull request to update this?

Reply:
Yes. I will update it.